WordPressVIPMinimum/Tests/Filters/RestrictedFilterUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+
+add_filter( 'test_filter', 'good_example_function' ); 	// this one shouldn't trigger anything
```

Comment:
Just an `// Ok.` is fine here as a comment.

Don't use tabs to separate the comment from the code - a single space is fine.

Remember full-stop, since that's needed as part of `WordPress-Docs`